### ISSUE TITLE
moralisAuth on Connect Button

### ISSUE THREAD

**Issue by PatrickAlphaC** [web3ui/web3uikit#275]
Right now if I have the following:

```
<ConnectButton moralisAuth={false} />
```

However I've added my server URL and appId, it still tries to have me do moralis authentication. Am I using the `moralisAuth` wrong? 

It says: 

https://github.com/web3ui/web3uikit/blob/9bb64014b8feb7aef2305d9fdcc5f4ca746048fe/src/components/ConnectButton/types.ts#L3

And I've set it to false, but it still tries to moralis auth?

**Comment by PatrickAlphaC** [comment]
Ah, if your provider looks like:

```
<MoralisProvider appId={APP_ID!} serverUrl={SERVER_URL!} initializeOnMount={true}>
```

It will auto try to moralis authenticate you. Is this intentional? Can we connect to moralis but not be authed as a moralis user? Or do we have to be authed?

**Comment by Y0moo** [comment]
fixed on 0.1.77
